mars/bash.cases

```diff
@@ -1205,7 +1206,7 @@
         </cmd>
     </case>
     <case>
-        <ignore>
+	<!--<ignore>
```

Comment:
why you add comment. if the bug is not correct then fix the bug number.

Reply:
No, it is correct for ASAP MLNX OFED 4.2 but I want to be able to run these tests on ofed 4.4. We don't want to add support for new project name now because we will do restructuring for project soon.

Reply:
so if you are going to ignore this forever just delete it.
if you are going to unignore to run on mlnx ofed 4.2 again then this ignore not relevant - maybe create bash_mofed42.cases or add another project MLNX_OFED_4_2 and create MLNX_OFED_4_4
you will have more versions later anyway so you probably will run on 2 versions (current & next) most of the time.

Reply:
The ignore is not forever, it will be removed once we did separate project for ofed 4.4 or create separate cases files for different project. We will work on this and it is in our plan but we are not going to start with it now. I don't want to keep ignoring them now on ofed 4.4 therefore I disabled these ignores and I will disable them via tags.